bin/sozo/src/commands/clean.rs

```diff
@@ -34,6 +41,7 @@ impl CleanArgs {
 
     pub fn run(self, config: &Config) -> Result<()> {
         let ws = scarb::ops::read_workspace(config.manifest_path(), config)?;
+        trace!(target: LOG_TARGET, "Workspace read successfully");
```

Comment:
Printing the workspace would be more helpful (if printable). If not we can remove this line.